assets/stylesheets/components/_crds-components-skeleton-blocks.scss

```diff
@@ -1,15 +1,15 @@
 .shared-header,
 .shared-header-skeleton {
-  min-height: 48px;
+  min-height: 49px;
```

Comment:
Did you do this? If so, why did we need to make the change – was there an issue?

Reply:
Yes there was issue when the shared-header loaded there was a 1px height jump when it switched from the shared-header-skeleton to the shared header